glx/single2.c

```diff
@@ -131,6 +133,8 @@ __glXDisp_RenderMode(__GLXclientState * cl, GLbyte * pc)
         goto noChangeAllowed;
     }
 
+    x_rpcbuf_t rpcbuf = { .swapped = client->swapped, .err_clear = TRUE };
```

Comment:
We need to move this up a little bit (to make sure that after the jump to `noChangeAllowed` our `rpcbuf` is initialized).

Reply:
indeed